src/common/providers/https.ts

```diff
@@ -661,6 +664,29 @@ function wrapOnCallHandler<Req = any, Res = any>(
       }
 
       const context: CallableContext = { rawRequest: req };
+
+      // This code is needed to fix v1 callable functions in the emulator with a monorepo setup.
+      // The original monkey-patched code lived in the functionsEmulatorRuntime
+      // (link: https://github.com/firebase/firebase-tools/blob/accea7abda3cc9fa6bb91368e4895faf95281c60/src/emulator/functionsEmulatorRuntime.ts#L480)
+      // and was not compatible with how monorepos separate out packages (see https://github.com/firebase/firebase-tools/issues/5210).
+      if (process.env.FUNCTIONS_EMULATOR && handler.length === 2) {
```

Comment:
nit* I'd prefer re-using `isDebugFeatureEnabled("skipTokenVerification")` logic instead of explicitly checking for existence of a super-special environment variable.

Reply:
SGTM, updated